Add German American Bank

DivyaB1 opened the pull request:
Hi,
I'm adding 'German American Bank' - https://germanamerican.com/personal/

bhousel commented on data/brands/amenity/bank.json:
```diff
@@ -21,6 +21,16 @@
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "bank",
+        "brand": "German American Bank",
```
```suggestion
        "brand": "German American",
```
Based on their branding, I would just call it "German American" in both the `brand` and `displayName` props.

<img width="300" alt="Screenshot 2023-07-21 at 12 30 07 PM" src="https://github.com/osmlab/name-suggestion-index/assets/38784/5ec740c0-da41-440f-afa9-6db478db9c1b">

bhousel commented on data/brands/amenity/bank.json:
```diff
@@ -21,6 +21,16 @@
   },
   "items": [
+     {
+      "displayName": "German American Bank",
```
```suggestion
      "displayName": "German American",
```

bhousel reviewed (changes requested):
Thanks!  What do you think of using the simpler name?  Their branding doesn't seem to include the word "Bank"

DivyaB1 commented:
Hi,
Thanks, they referred themselves as 'German American Bank' in their 'About Us' page so I added it. I've corrected it now!
https://germanamerican.com/about/history/#:~:text=The%20history%20of-,German%20American%20Bank,-tells%20quite%20an

bhousel commented:
Thanks @DivyaB1 👍
